content/applications/finance/payment_providers.rst

```diff
@@ -169,246 +162,267 @@ Bank payments
 
-.. note::
-   All prices shown in the express checkout payment form are always taxes included.
+.. _payment_providers/features/tokenization:
 
-.. _payment_providers/configuration:
+Tokenization
+============
 
-Configuration
-=============
+:ref:`If the payment provider supports this feature <payment_providers/online_providers>`, customers
+can choose to save their card details for later. To enable this feature, go to the
+:guilabel:`Configuration` tab of the selected payment provider and enable :guilabel:`Allow Saving
+Payment Methods`.
 
-.. note::
-   Each provider has its specific configuration flow, depending on :ref:`which feature is available
-   <payment_providers/online_providers>`.
+In this case, a **payment token** is created in Odoo to be used as a payment method for subsequent
+payments, without the customer having to enter their card details again. This is particularly useful
+for the eCommerce conversion rate and for subscriptions that use recurring payments.
 
-.. _payment_providers/add_new:
+.. note::
+   You remain fully PCI-compliant when you enable this feature because Odoo does not store the card
+   details directly. Instead, it creates a payment token that only holds a reference to the card
+   details stored on the payment provider's server.
 
-Add a new payment provider
---------------------------
+.. _payment_providers/features/manual_capture:
 
-To add a new payment provider and make it available to your customers, go to
-:menuselection:`Accounting --> Configuration --> Payment Providers`, look for your payment provider,
-install the related module, and activate it. To do so, open the payment provider and change its
-state from *Disabled* to *Enabled*.
+Manual capture
+==============
 
-.. image:: payment_providers/activation.png
-   :align: center
-   :alt: Click on install, then on activate to make the payment provider available on Odoo.
+:ref:`If the payment provider supports this feature <payment_providers/online_providers>`, you can
+authorize and capture payments in two steps instead of one. To enable this feature, go to the
+:guilabel:`Configuration` tab of the selected payment provider and enable :guilabel:`Capture Amount
+Manually`.
 
-.. warning::
-   We recommend using the *Test Mode* on a duplicated database or a test database. The Test Mode is
-   meant to be used with your test/sandbox credentials, but Odoo generates Sales Orders and Invoices
-   as usual. It isn't always possible to cancel an invoice, and this could create some issues with
-   your invoices numbering if you were to test your payment providers on your main database.
+When you authorize a payment, the funds are reserved on the customer's payment method but they are
+not immediately charged. The charge is only made when you manually capture the payment later on.
+You can also void the authorization to release the reserved funds; this is equivalent to a regular
+cancellation. Capturing payments manually is useful in many situations:
```

Comment:
Cancellations are more of a customer action than a vendor action. Vendors typically have to refund payments that they want to revert when they have already been captured. The difference with a void is that a void releases the funds by canceling the authorization before the capture moves any money. In practice, you don't have to pay expensive (several percent) refund fees.

https://support.stripe.com/questions/understanding-fees-for-refunded-payments

Reply:
See third bullet point.